src/__test__/components/data-exploration/embedding/Embedding.test.jsx

```diff
@@ -12,12 +12,12 @@ import preloadAll from 'jest-next-dynamic';
 // eslint-disable-next-line import/extensions
 import { Scatterplot } from 'vitessce/dist/es/production/scatterplot.min.js';
 import Embedding from '../../../../components/data-exploration/embedding/Embedding';
-import CrossHair from '../../../../components/data-exploration/embedding/CrossHair';
-import CellInfo from '../../../../components/data-exploration/CellInfo';
-import { CELL_SETS_CREATE } from '../../../../redux/actionTypes/cellSets';
+// import CrossHair from '../../../../components/data-exploration/embedding/CrossHair';
+// import CellInfo from '../../../../components/data-exploration/CellInfo';
+// import { CELL_SETS_CREATE } from '../../../../redux/actionTypes/cellSets';
```

Comment:
why are all these commented out?

Reply:
if they are not needed, just delete them

Reply:
I first deleted them, but later realized that there were some tests commented tests that probably would use them if uncommented. So I just went for removing the warnings rather than finding out more about those tests.

Reply:
wow, you are right. Why were these tests commented?

Reply:
And how did we let them to master commented out?

Reply:
idk
https://github.com/biomage-ltd/ui/pull/126

Reply:
Yeah, I can see how it happened -- github didn't expand the diff and the file never got checked. Could you try to uncomment those tests and fix them as part of this PR?

Reply:
Those test do not fail... by now, because this warning becomes an error starting on node 15. They all get

```
(node:29173) UnhandledPromiseRejectionWarning: Error: Failed to create WebGL context
(node:29173) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

I tried with https://www.npmjs.com/package/jest-webgl-canvas-mock or https://www.npmjs.com/package/webgl-mock but it did not work.

(see https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1614195847032500?thread_ts=1614181629.017500&cid=C015CHSUDRU)

Reply:
I see. We will need to fix them eventually if we want to move to node v 15. I will make a separate ticket for fixing the tests, you can leave it for now.